src/main/scala/org/constellation/ConfigUtil.scala

```diff
@@ -57,6 +57,12 @@ object ConfigUtil {
   def isEnabledAWSStorage: Boolean =
     Try(configStorage.getBoolean("aws.enabled")).getOrElse(false)
 
+  def isEnabledAWSBackupStorageA: Boolean =
+    Try(configStorage.getBoolean("aws-backup-a.enabled")).getOrElse(false)
+
+  def isEnabledAWSBackupStorageB: Boolean =
+    Try(configStorage.getBoolean("aws-backup-b.enabled")).getOrElse(false)
+
```

Comment:
I wonder if maybe we should also be checking here if isEnabledAWSStorage is true and then we could load aws-access-key and aws-secret-key from that configuration. I guess for our case we will not have different secret and key for different buckets.

Reply:
We definitely won't have  separated keys for different buckets, so I just suggest you to pass buckets as array in `application.conf`